process_manager/views/partials.py

```diff
@@ -50,17 +51,18 @@ def filter_table(
     return [row for row in table if any(search in str(row[k]).lower() for k in columns)]
 
 
+@sync_to_async
 @login_required
 @handle_errors
-def process_table(request: HttpRequest) -> HttpResponse:
+@async_to_sync
+async def process_table(request: HttpRequest) -> HttpResponse:
```

Comment:
Assuming I got this right from the [Django docs on the topic](https://docs.djangoproject.com/en/5.1/topics/async/), we need this stack of decorators because `@login_required` and `@handle_errors` need to run in the main event loop, right? But except that bit - before entering into the view or if getting out of it unexpectedly because of an error - the view itself runs asyncronously. Is this correct?